helm_deploy/values-preprod.yaml

```diff
@@ -0,0 +1,32 @@
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 4
```

Comment:
I wonder what the advantages are of having more than 1 assuming health check works ok - since it doesn't matter if we have 1 minutes outrage during restarts?

On balance I kind of think we should only have 1 but more than happy to see if having mutilples causes any issues

@Mjwillis What's your view?

Reply:
well cloud platform want us normally to run 4 replicas for services, for when they recycle nodes and the such like.  @Mjwillis has reported no issues with keyworker-api since that is retrieving messages from queues too